src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -759,7 +759,8 @@ ConvSolution InitInvokerFactoryNCHW(const ExecutionContext& ctx,
             /// \todo: Will need SetTensor() to properly zero out non-packed tensors
             /// Note: Need to clear buffer memory for BWD data since all values may not be set.
             if(output_tr_inst.GetConvOperandTag() == internal::ConvOperandTag::Weights ||
-               output_tr_inst.GetConvOperandTag() == internal::ConvOperandTag::Input)
+               output_tr_inst.GetConvOperandTag() == internal::ConvOperandTag::Input ||
+               output_tr_inst.GetConvOperandTag() == internal::ConvOperandTag::Output)
```

Comment:
Are there other operands besides these three?  If not, can we just remove the if statement at all and just always zero out the buffer and say in a comment that it's needed in all cases?